appengine/flexible/scipy/main.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+from flask import Flask
+import scipy.misc
+
+app = Flask(__name__)
+
+
+# [START scipy]
+@app.route('/')
+def resize():
+    """Demonstrates using scipy to resize an image."""
+    app_path = os.path.dirname(os.path.realpath(__file__))
+    image_path = os.path.join(app_path, 'assets/google_logo.jpg')
+    img = scipy.misc.imread(image_path)
+    img_tinted = scipy.misc.imresize(img, (300, 300))
+    output_image_path = os.path.join(
+        app_path, 'assets/resized_google_logo.jpg')
+    # Write the tinted image back to disk
+    scipy.misc.imsave(output_image_path, img_tinted)
+    return "Image resized."
```

Comment:
I don't think we should really be serving static images from the filesystem like that and I'd rather keep it simple than demonstrate a bad practice. I'm more inclined to actually upload to a bucket but it just seems like overkill considering the only point is to say "hey, scipy will work if you add it to the requirements.txt"